### PR TITLE
Add Apple TV IPA upload support for Transporter

### PR DESCRIPTION
**Pull request by sunboshan**
This will resolve issue 
- https://github.com/fastlane/pilot/issues/55
- https://github.com/fastlane/pilot/issues/92

Tested already.

### CODE REVIEW

File: lib/fastlane_core/ipa_upload_package_builder.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ def generate(app_id: nil, ipa_path: nil, package_path: nil)
         file_size: File.size(ipa_path),
         ipa_path: File.basename(ipa_path), # this is only the base name as the ipa is inside the package
         md5: Digest::MD5.hexdigest(File.read(ipa_path))
+        platform: platform || "ios" # pass "appletvos" for Apple TV's IPA
```
Comment (KrauseFx) [inline lib/fastlane_core/ipa_upload_package_builder.rb]: Could you surround this with `(platform || "ios")`?

Comment (sunboshan) [inline lib/fastlane_core/ipa_upload_package_builder.rb]: Sorry missed that part! It's done. :)

**Comment by KrauseFx** [comment]
Thank you so much @sunboshan :rocket:

**Comment by tmm1** [comment]
> This will resolve issue
> 
> fastlane/pilot#55
> fastlane/pilot#92

How does this fix the issues in pilot? AFAICT, pilot is not using IpaUploadPackageBuilder

**Comment by KrauseFx** [comment]
Nice catch @tmm1, `pilot` should be updated to use the same class

**Pull request merged**